modules/cluster-logging-upgrading-logging.adoc

```diff
@@ -61,36 +61,36 @@ by Red Hat, Inc
 
 .. Click *Subscription* -> *Channel*.
 
-.. In the *Change Subscription Update Channel* window, select *4.3* and click *Save*.
+.. In the *Change Subscription Update Channel* window, select *4.4* and click *Save*.
 
 .. Wait for a few seconds, then click *Operators* -> *Installed Operators*.
 +
-The Cluster Logging Operator is shown as 4.3. For example:
+The Cluster Logging Operator is shown as 4.4. For example:
 +
 ----
 Cluster Logging
-4.3.0-201909201915 provided
+4.4.0-201909201915 provided
 by Red Hat, Inc
 ----
 
 . Check the logging components:
 
-.. Ensure that the Elasticsearch Pods are using a 4.3 image:
+.. Ensure that the Elasticsearch Pods are using a 4.4 image:
 +
 ----
 $ oc get pod -o yaml -n openshift-logging --selector component=elasticsearch |grep 'image:'
 
-image: registry.redhat.io/openshift4/ose-logging-elasticsearch5:v4.3.0-202001081344
+image: registry.redhat.io/openshift4/ose-logging-elasticsearch5:v4.4.0-202001081344
+image: registry.redhat.io/openshift4/ose-oauth-proxy:v4.4.0-202001081344
+image: registry.redhat.io/openshift4/ose-logging-elasticsearch5:v4.4.0-202001081344
 image: registry.redhat.io/openshift4/ose-oauth-proxy:v4.3.0-202001081344
```

Comment:
This list looks off because there is `ose-oauth-proxy:v4.3.0` still listed but you added a `ose-oauth-proxy:v4.4.0` above

Reply:
I can look into these

Reply:
Thank you!